docs/UserGuide.md

```diff
@@ -206,6 +206,14 @@ if it is open. </div>
 In order to maintain the database, we have provided several basic commands such as `add`, `edit`, `delete` and `clear`
 to help you modify resident data within **RC4HDB**.
 
+Note:
+* **RC4HDB** does not allow duplicate residents to exist within the database, as a measure to prevent unintentional adding of duplicate residents.
+* Two residents are considered duplicate if they have the same:
+  * matriculation number
+  * phone number
+  * email
+  * room
+
```

Comment:
I think we can make it clearer here by adding "any of the following" right before the fields are listed.